config/config.go

```diff
@@ -255,6 +255,7 @@ func NewConfigForTests() *Config {
 	c.DisableSecurityProviderVerify = true
 	c.LogFile = "discard"
 	c.RPCAuthorization = false
+	c.Choria.FederationCollectives = []string{"alpha", "beta"}
```

Comment:
Couldn't figure out how to get things to work via os.Setenv :\

Reply:
go has some issues getting environment in containers, i dont know we had another case of it not working reported and I couldnt figure it out.

Reply:
Do you want me testing for the case where federation is not setup?

Reply:
Also what was weird was that I could do os.Setenv(), and then do os.Getenv() and retrieve the value, but fetching it out from fw.FederationCollectives wouldn't do the right thing.  I'll take a longer look at it.

Reply:
Figured it out, it does work.  Need to setup the mocks though.